phpunit.xml.dist

```diff
@@ -11,7 +11,7 @@
     </testsuites>
 
     <php>
-        <server name="KERNEL_CLASS_PATH" value="/tests/Application/AppKernel.php" />
+        <server name="KERNEL_CLASS_PATH" value="/tests/Application/Kernel.php" />
```

Comment:
The corresponding file is not changed. It was changed before ?

Reply:
Yes, but not change here ;( So i fixed it.